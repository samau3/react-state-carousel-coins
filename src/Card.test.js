import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

test("renders without crashing", function () {
    render(<Card caption="Test" currNum={1} src="test.png" totalNum={3} />);
});


test("matches snapshot", function () {
    const { container } = render(<Card caption="Test" currNum={1} src="test.png" totalNum={3} />);
    expect(container).toMatchSnapshot();
});