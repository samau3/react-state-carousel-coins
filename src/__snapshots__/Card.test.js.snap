// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Test
    </h4>
    <img
      alt="Test"
      class="Card-image"
      src="test.png"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
